addons/event/report/event_event_templates.xml

```diff
@@ -280,7 +291,28 @@
     <div class="o_event_badge_ticket_wrapper" t-att-style="'background-image: url(%s);' % badge_image_url if badge_image_url else ''">
         <div class="position-relative h-100">
             <h3 class="fw-bold text-center" t-field="event.name"/>
-            <div class="text-center mt-5 py-2">
+            <div class="text-center o_event_badge_font_small">
+                <span itemprop="startDate" t-field="event.date_begin"
+                    t-options='{"widget": "datetime", "date_only": True}'
+                    class="fw-bold"/>
+                <span itemprop="startDateTime" t-field="event.date_begin"
+                    class="fw-bold"
+                    t-options='{"widget": "datetime", "time_only": True, "tz_name": event.date_tz, "hide_seconds": True}'/>
+                <span class="fa fa-arrow-right o_event_badge_font_faded"/>
+                <span t-if="not event.is_one_day"
+                    itemprop="endDate" t-field="event.date_end"
+                    t-options='{"widget": "datetime", "date_only": True}'
+                    class="fw-bold"/>
+                <span itemprop="endDateTime" t-field="event.date_end"
+                    class="fw-bold"
+                    t-options='{"widget": "datetime", "time_only": True, "tz_name": event.date_tz, "hide_seconds": True}'/>
+            </div>
+            <div t-if="event.address_id" class="o_event_badge_font_faded o_event_badge_font_small text-center">
+                <t t-call="event.event_report_template_formatted_event_address">
```

Comment:
Ok I see why we have "use_line_breaks" now ^^

Trying to find a more meaningful name... "multi_line_address" maybe?